### PR TITLE
install: Allow forwarding options to `docker build`

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
@joseph-wakeling-sociomantic this might come handy for the mnist DB.

I will include it in v0.1.2 because it's almost a bug that we couldn't pass arguments in the past :D

**Comment by leandro-lucarella-sociomantic** [comment]
Argh, damn, I did this a long long ago and forgot to send the PR, this was about the `build`, not `run`. I will fix.

**Comment by leandro-lucarella-sociomantic** [comment]
@joseph-wakeling-sociomantic nevermind, you can pass arguments to the `beaver run` in the current version.

**Comment by joseph-wakeling-sociomantic** [comment]
Your choice of milestone seems off: 0.1.2 is already released!

**Comment by leandro-lucarella-sociomantic** [comment]
Oh, damn!

**Comment by leandro-lucarella-sociomantic** [comment]
OK, I will auto-merge, release v0.1.3 and move to work in v0.2.0.

File: bin/beaver-install [hunk 1]
```diff
@@ -7,4 +7,4 @@
 d="$(dirname "$0")"
 
 # Build the docker image using .$DIST as suffix if present
-"$d/beaver" docker build -f "Dockerfile${DIST:+.$DIST}" .
+"$d/beaver" docker build -f "Dockerfile${DIST:+.$DIST}" "$@" .
```
Comment (nemanja-boric-sociomantic) [inline bin/beaver-install:10]: this looks fine.

**Pull request merged**